docs/man_pages/general/logout.md

```diff
@@ -1,20 +1,18 @@
 logout
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
 General | `$ appbuilder logout`
+
 Logs you out from the Telerik Platform.
-To build, deploy, or synchronize your projects with the cloud, you need to be logged in.
-<% if(isHtml) { %> 
 
-#### Related Commands
+<% if(isConsole) { %>IMPORTANT: To build, deploy, or synchronize your projects with the cloud, you need to be logged in.<% } %>
```

Comment:
Should we also add "Commands which require the cloud services will automatically log you in"?